### ISSUE TITLE
user names in old-style reply count towards character limit

### ISSUE THREAD

**Issue by kathakatze** [eramdam/BetterTweetDeck#150]
The new reply style enables replies with 140 characters, but enabling the old style will subtract all user names from this limit, making long replies impossible.

**Comment by eramdam** [comment]
That's how it works unfortunately. All the other clients that allow to switch from one to the other have the same issue and I cannot do anything about that. Sorry!

**Comment by kathakatze** [comment]
Twidere for Android manages this issue pretty well and treats user names at the beginning of the tweet as some sort of virtual text. I don't know anything about tweetdeck's internals, so I can't judge whether handling replies the same way is possible. The comparison could be a bit unfair because Twidere has the advantage of controlling every aspect.

**Comment by eramdam** [comment]
Exactly. I don't think it would be safe to tinker the internals of TweetDeck that much. We'll see if at some point in the future I manage to make this work but since the "old style replies" are meant to disappear in the future anyway, I don't see this as a very useful dev 😅
